static-assets/components/cstudio-common/common-api.js

```diff
@@ -6224,12 +6224,11 @@ var nodeOpen = false,
         $(document).on('click', `.notifyjs-${styleName}-${id} .yes`, onOk);
       },
 
-      showConfirmDialog: function (title, body, callback, okButtonText, cancelButtonText) {
-        const onOk = 'confirmDialogOnOk';
-        const onCancel = 'confirmDialogOnCancel';
-        let unsubscribe, cancelUnsubscribe;
+      showConfirmDialog: function ({ title, body, onOk, onCancel, okButtonText, cancelButtonText }) {
```

Comment:
Make this change backward compatible, add to changelog and test all affected usages
```suggestion
      showConfirmDialog: function (
        // title, body, callback, okButtonText, cancelButtonText
        // { title, body, onOk, onCancel, okButtonText, cancelButtonText }
      ) {
        const { title, body, onOk, onCancel, okButtonText, cancelButtonText } = typeof arguments[0] === 'object' ? arguments[0] : { title: arguments[a], body: arguments[b], etc... }
```